routing.go

```diff
@@ -399,26 +393,18 @@ func (dht *IpfsDHT) getValues(ctx context.Context, key string, stopQuery chan st
 		if err != nil {
 			return
 		}
-		queriesCh <- queries
+		lookupResCh <- lookupRes
 
 		if ctx.Err() == nil {
```

Comment:
I suspect we could bundle this check into the `lookup.completed` variable, but haven't gotten around to it yet.